app/Models/User.php

```diff
@@ -118,6 +118,10 @@ public function fullName()
         return "{$this->first_name} {$this->last_name}";
     }
 
+    public function getFullNameAttribute()
```

Comment:
Functionally it's the same, the difference is that laravel magic makes a  'full_name' property available as a value in an Eloquent Collection this way, so things like lists() function.  We could either make getDetailedNameAttribute() return fullName(), or modify everywhere that uses ->fullName() currently to instead call ->full_name, and remove fullName().  I'm not sure which is a better approach.